plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerLauncher.java

```diff
@@ -94,6 +100,10 @@ public void sendProgressReport(ProgressReport report) {
     processorJsonRpcCommunication.sendProgressNotification(report);
   }
 
+  public CompletableFuture<Object> executeClientCommand(ExecuteCommandParams params) {
+    return executeCliendCommandTransmitter.executeClientCommand(params);
```

Comment:
This is a place where we could recognize specific commands and transform file:///projects uri's.